website/docs/r/project.html.markdown

```diff
@@ -31,6 +31,8 @@ The following arguments are supported:
 
 * `path` - (Optional) The path of the repository.
 
+* `path_with_namespace` - The path of the repository with namespace.
```

Comment:
Can you move this to the `Attribute Reference` section of this page?

Reply:
Sure ✅ 

Didn't pay attention to that 🤦